unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/DefaultChatEntry.cs

```diff
@@ -0,0 +1,323 @@
+using System;
+using System.Collections.Generic;
+using DCL;
+using DCL.Helpers;
+using DCL.Interface;
+using DCL.SettingsCommon;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class DefaultChatEntry : ChatEntry, IPointerClickHandler, IPointerEnterHandler, IPointerExitHandler
+{
+    [SerializeField] internal float timeToFade = 10f;
+    [SerializeField] internal float fadeDuration = 5f;
+    [SerializeField] internal TextMeshProUGUI body;
+    [SerializeField] CanvasGroup group;
+    [SerializeField] internal float timeToHoverPanel = 1f;
+    [SerializeField] internal float timeToHoverGotoPanel = 1f;
+    [SerializeField] private bool showUserName = true;
+    [SerializeField] private RectTransform hoverPanelPositionReference;
+    [SerializeField] private RectTransform contextMenuPositionReference;
+    [NonSerialized] public string messageLocalDateTime;
+
+    private bool fadeEnabled;
+    private double fadeoutStartTime;
+    private float hoverPanelTimer;
+    private float hoverGotoPanelTimer;
+    private bool isOverCoordinates;
+    private ParcelCoordinates currentCoordinates;
+    private ChatEntryModel model;
+
+    public override ChatEntryModel Model => model;
+
+    public event Action<string> OnPress;
+    public event Action<DefaultChatEntry> OnPressRightButton;
+    public event Action<DefaultChatEntry> OnTriggerHover;
+    public event Action<DefaultChatEntry, ParcelCoordinates> OnTriggerHoverGoto;
+    public event Action OnCancelHover;
+    public event Action OnCancelGotoHover;
+
+    public override void Populate(ChatEntryModel chatEntryModel)
+    {
+        model = chatEntryModel;
+
+        string userString = GetDefaultSenderString(chatEntryModel.senderName);
+
+        if (chatEntryModel.messageType == ChatMessage.Type.PRIVATE)
+        {
+            if (chatEntryModel.subType == ChatEntryModel.SubType.RECEIVED)
+            {
+                userString = $"<b><color=#5EBD3D>From {chatEntryModel.senderName}:</color></b>";
+            }
+            else if (chatEntryModel.subType == ChatEntryModel.SubType.SENT)
+            {
+                userString = $"<b>To {chatEntryModel.recipientName}:</b>";
+            }
```

Comment:
i dont really find an improvement here, since the format is no repeated in many places.. this is how the code would look like, which i think it is less understandable:
```
private const string RECEIVED_WHISPER_FORMAT = "<b><color=#5EBD3D>From {0}:</color></b>";
....
userString = string.Format(RECEIVED_WHISPER_FORMAT, chatEntryModel.senderName);
```